src/canmatrix/formats/arxml.py

```diff
@@ -1560,6 +1563,8 @@ def get_frame(frame_triggering, ea, multiplex_translation, float_factory, header
             pdu = ipdu[0]
 
         new_frame = canmatrix.Frame(ea.get_element_name(frame_elem), size=int(dlc_elem.text, 0))
+        new_frame.add_pdu(target_pdu)
```

Comment:
Hi @xRowe 

`target_pdu` is not defined here, this leads to an error in my tests.
Could you please have another look on it?

Thanks